lib/OnyxCache.js

```diff
@@ -110,7 +111,7 @@ class OnyxCache {
      * @param {Record<string, *>} data - a map of (cache) key - values
      */
     merge(data) {
-        this.storageMap = mergeWithCustomized({}, this.storageMap, data);
+        this.storageMap = Object.assign({}, merge(this.storageMap, data));
```

Comment:
I think you can suppress the lint errors here. If we want this to be fast, we shouldn't use the spread operator.